### PR TITLE
feat: dae can call now

### PR DESCRIPTION
**Pull request by troubadour-hell**
### Background

As a dae, it should call.

### Checklist

- [ ] The Pull Request has been fully tested
- [ ] There's an entry in the CHANGELOGS
- [ ] There is a user-facing docs PR against https://github.com/daeuniverse/dae

### Full Changelogs

- [Implement ...]

### Issue Reference

Closes #_[issue number]_

### CODE REVIEW

**Comment by mzz2017** [comment]
@troubadour-hell Use 'honk' instead?

**Comment by troubadour-hell** [comment]
> 

Done.

**Comment by troubadour-hell** [comment]
> @troubadour-hell Use 'honk' instead?

done.

File: cmd/call.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	 "github.com/spf13/cobra"
+ )
+ 
+ var (
+	 honkCmd = &cobra.Command{
+		 Use:   "honk",
+		 Short: "Let dae call for you.",
+		 Run: func(cmd *cobra.Command, args []string) {
+				 fmt.Println("Honk! Honk! Honk! This is dae!")
+				 os.Exit(1)
```
Comment (mzz2017) [inline cmd/call.go:21]: 1 indicates errors. We should use os.Exit(0) here.

BTW, do you have tested it? Please paste the result. Thank you.

**Review by mzz2017 — CHANGES_REQUESTED** [review]

File: cmd/call.go (hunk 1 above)
Comment (troubadour-hell) [reply cmd/call.go]: With exit(1), it works like:
image
And it was changed to exit(0) in this fix commit.

**Comment by mzz2017** [comment]
Interesting PR. Thank you!

**Review by mzz2017 — APPROVED** [review]

**Pull request merged**